cockatrice/src/chatview/chatview.cpp

```diff
@@ -12,10 +12,17 @@
 #include <QDesktopServices>
 #include <QMouseEvent>
 #include <QScrollBar>
+#include <QTextDocumentFragment>
 #include <QTextEdit>
 
 const QColor DEFAULT_MENTION_COLOR = QColor(194, 31, 47);
 
+UserMessagePosition::UserMessagePosition(QTextCursor &cursor)
+{
```

Comment:
What does this function do? I'm confused

Reply:
it's the constructor for the message position, it saves the block and the position in the block it references